src/WebAppDIRAC/Lib/WebHandler.py

```diff
@@ -369,3 +369,11 @@ def on_open(self):
     def _getMethod(self):
         """Get method function to call."""
         return self.on_open
+
+    def on_message(self, msg):
+        """Setup the threadConfig before doing the actual calls.
+            Developper should implement ``_on_message`` instead
```

Comment:
```suggestion
            Developer should implement ``_on_message`` instead
```
and missing an empty line below.